framework/console/controllers/BaseMigrateController.php

```diff
@@ -61,7 +61,12 @@
      * @since 2.0.7
      */
     public $fields = [];
-
+    /**
+     * @var Connection|array|string the DB connection object or the application component ID of the DB connection to use
+     * when applying migrations. Starting from version 2.0.3, this can also be a configuration array
+     * for creating the object.
+     */
+    public $db = 'db';
```

Comment:
That is wrong because `BaseMigrateController` is used, for example, for MongoDB which does not have prefixes.